libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorImageSettingsListener.java

```diff
@@ -2,4 +2,5 @@
 
 public interface EditorImageSettingsListener {
     void onImageSettingsRequested(EditorImageMetaData editorImageMetaData);
+    void onAudioSettingsRequested(String mediaModelId);
```

Comment:
let's remove this method from this interface (as a side note, this interface is called `EditorImageSettingsListener`, so it doesn't indicate it has a relationship with `audio`)